### DISCUSSION TITLE
Thanks for Immich!

### DISCUSSION THREAD

**Discussion by muety** [immich-app/immich#11097] [General]
Just quickly wanted to say thanks to the maintainers and all contributors. To me, Immich is perfect in literally every aspect and the photos app that I've been looking for for long. Keep up the good work!

**Comment by elmuziko** [comment]
I second this. 
I intend to build a second UNRAID nas before the summer is out, once I figure out how to give Immich full DR coverage over the two boxes I'm reducing my iCloud plan. Immich is far superior than Apple photos could ever be.
